fieldservice_sale_recurring/readme/USAGE.md

```diff
@@ -0,0 +1,7 @@
+- Go to Sales
+- Create a new Quotation/Sale Order
+- Set the FSM Location to be used
+- On a Sale Order Line, select a product configured for field service
+  orders
+- Confirm the Sale Order
+- Field Service Orders linked to SO lines are created
```

Comment:
I don't think this explanation is the one for fieldservice_sale_recurring. Can you update the USAGE ?